src/Webscrape/WebscrapePage.jsx

```diff
@@ -0,0 +1,36 @@
+import React, {useState} from "react";
+import Input from "../Input";
+import { submitNewURL } from "../api";
+import { withCookies } from "react-cookie";
+import { useHistory } from "react-router-dom";
+
+const WebscrapePage = ({ allCookies = {} }) => {
```

Comment:
if you're using `allCookies` prop you need to wrap the export default with the `withCookies`